Action/vrijbrp.ZgwToVrijbrpSOAPGeheimhoudingAction.action.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "ZgwToVrijbrpSOAPGeheimhoudingAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.ZgwToVrijbrpSOAPGeheimhoudingAction.action.json",
+  "$schema": "https://json-schema.org/draft/2020-12/action",
+  "version": "0.0.2",
+  "listens": [
+    "vrijbrp.zaak.created"
+  ],
+  "throws": [],
+  "conditions": {
+    "==": [
+      {"var": "object.zaaktype.identificatie"},
+      "B0328"
+    ]
+  },
+  "class": "CommonGateway\\NaamgebruikVrijBRPBundle\\ActionHandler\\ZgwToVrijbrpGeheimhoudingActionHandler",
+  "configuration": {
```

Comment:
location is required in the ActionHandler configuration, so it should be present or not required